### PR TITLE
Restructure composite models folders

### PR DESCRIPTION
**Pull request by Anna-Mironova**
This pull request duplicates https://github.com/opencv/open_model_zoo/pull/903 and contains the combined descriptions for piplined models and updated index.md files.

### CODE REVIEW

File: models/intel/text-spotting-0002/description/text-spotting-0002.md [hunk 1]
```diff
@@ -48,5 +49,53 @@ This model is a Mask-RCNN-based text detector with ResNet50 backbone and additio
...
 5.  Name: `text_features`, shape [100x64x28x28]. Text features that are fed to a text recognition head.
 
+
+## Encoder model specification
+
+The text-spotting-0002-recognizer-encoder model is a fully-convolutional encoder of text recognition head.
+
+| Metric                                        | Value     |
+|-----------------------------------------------|-----------|
+| Word spotting hmean ICDAR2015, without a dictionary | 59.04%    |
```
Comment (eaidova) [inline models/intel/text-spotting-0002/description/text-spotting-0002.md:59]: on my view, this metric does not correspond to model part, it is accuracy of whole pipeline, so should be moved to level up

Comment (eaidova) [reply models/intel/text-spotting-0002/description/text-spotting-0002.md:59]: possibly framework and type too relates to whole model

Comment (Anna-Mironova) [reply models/intel/text-spotting-0002/description/text-spotting-0002.md]: Added part composite model specification

File: models/intel/action-recognition-0001/description/action-recognition-0001.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+# Composite model name: action-recognition-0001
```
Comment (IRDonch) [inline models/intel/action-recognition-0001/description/action-recognition-0001.md:1]: This should be consistent with the non-composite models. I would suggest something like:

```markdown
# action-recognition-0001 (composite)
```

File: models/intel/action-recognition-0001/description/action-recognition-0001.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+## Use Case and High-Level Description
+
+This is a general-purpose action recognition composite model, consisting from encoder and decoder parts, trained on Kinetics-400 dataset. The encoder model uses Video Transformer approach with ResNet34 encoder.
```
Comment (IRDonch) [inline models/intel/action-recognition-0001/description/action-recognition-0001.md:5]:
```suggestion
This is a general-purpose action recognition composite model, consisting of encoder and decoder parts, trained on Kinetics-400 dataset. The encoder model uses Video Transformer approach with ResNet34 encoder.
```

File: models/intel/text-spotting-0002/description/text-spotting-0002.md [hunk 4]
```diff
@@ -1,42 +1,49 @@
...
+This is a text spotting composite model that simultaneously detects and
 recognizes text. The model detects symbol sequences separated by space and performs
 recognition without a dictionary. The model is built on top of the Mask-RCNN
 framework with additional attention-based text recognition head.
 
 Symbols set is alphanumeric: `0123456789abcdefghijklmnopqrstuvwxyz`.
 
-This model is a Mask-RCNN-based text detector with ResNet50 backbone and additional text features output.
+*Hmean Word spotting* is defined and measured according to the
+[Incidental Scene Text (ICDAR2015) challenge](https://rrc.cvc.uab.es/?ch=4&com=introduction).
```
Comment (IRDonch) [inline models/intel/text-spotting-0002/description/text-spotting-0002.md:13]: This seems like it should be in the "Composite model specification" section, not here.

File: models/intel/action-recognition-0001/description/action-recognition-0001.md (hunk 2 above)
Comment (Anna-Mironova) [reply models/intel/action-recognition-0001/description/action-recognition-0001.md]: Done for all models

File: models/intel/action-recognition-0001/description/action-recognition-0001.md (hunk 3 above)
Comment (Anna-Mironova) [reply models/intel/action-recognition-0001/description/action-recognition-0001.md]: Done

File: models/intel/text-spotting-0002/description/text-spotting-0002.md (hunk 4 above)
Comment (Anna-Mironova) [reply models/intel/text-spotting-0002/description/text-spotting-0002.md]: Done

**Pull request merged**